### PR TITLE
TC_006.002.003 I What's new > Eco Collection New* > Redirection to the product page by clicking on the image

### PR DESCRIPTION
**Pull request by ElenaBarteneva**
<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;aj3EgeOa">TC_006.002.003 I What&#39;s new &gt; Eco Collection New* &gt; Redirection to the product page by clicking on the image</a></blockquote>

### CODE REVIEW

**Comment by ElenaBarteneva** [comment]
[TC_006.002.003 I What's new > Eco Collection New* > Redirection to the product page by clicking on the image](https://trello.com/c/aj3EgeOa/281-tc006002003-i-whats-new-eco-collection-new-redirection-to-the-product-page-by-clicking-on-the-image)

File: pages/whats_new_page.py [hunk 1]
```diff
@@ -85,6 +85,23 @@ def is_yoga_link_visible(self):
...
 
+
+    def open_eco_collection_url(self):
+        self.open_page()
+        s(BaseLocators.ECO_COLLECTION_NAME).click()
+
+    def click_layla_tee_name(self):
+        s(Product.LAYLA_TEE_PRODUCT_NAME).click()
+
+    def check_redirection_to_layla_tee_pdp(self):
```
Comment (Lokankara) [inline pages/whats_new_page.py:96]: what it means `pdp`?
name of method can be like `is_current_url_layla_tee_product_detail_page`

File: pages/whats_new_page.py [hunk 2]
```diff
@@ -85,6 +85,23 @@ def is_yoga_link_visible(self):
...
+        return self.get_current_url() == LAYLA_TEE_URL
+
+    def layla_tee_title_is_displayed(self):
```
Comment (Lokankara) [inline pages/whats_new_page.py:99]: name of method should start with verb:
`verify_layla_tee_title_displayed`

File: tests/test_whats_new.py [hunk 3]
```diff
@@ -53,6 +53,35 @@ def test_lumas_latest_list_visibility(self):
         assert item_number == 4
         assert page.are_men_and_women_items_present() is True
 
+
+    @allure.title(
+        "TC_006.002.004 I What's new > Eco Collection New* > Redirection to the product page by clicking on the product name")
+    @allure.link('https://trello.com/c/GO8VRlcn')
+    def test_eco_collection_redirection_to_pdp(self, login, browser_management):
```
Comment (Lokankara) [inline tests/test_whats_new.py:60]: Parameters 'login' and  'browser_management' value is not used

File: tests/test_whats_new.py [hunk 4]
```diff
@@ -53,6 +53,35 @@ def test_lumas_latest_list_visibility(self):
...
+        with allure.step('Opening Eco Collection New page'):
+            page = WhatsNewPage(browser=browser)
+            page.open_eco_collection_url()
+        with allure.step('Opening Layla Tee page'):
+            page.click_layla_tee_name()
```
Comment (Lokankara) [inline tests/test_whats_new.py:65]: you have at least 3 pages:  WhatsNewPage,  Eco Collection page,  Layla Tee page
and test should be like this:
`eco_collection_page = page.open_eco_collection()`
`layla_tee_page = eco_collection.page.open_layla_tee() `

**Review by Lokankara — APPROVED** [review]

**Pull request merged**